CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 ### Bug Fixes
 
 * **deps:** bumped dependencies versions to resolve CVEs ([#13789](https://github.com/grafana/loki/issues/13789)) ([34206cd](https://github.com/grafana/loki/commit/34206cd2d6290566034710ae6c2d08af8804bc91))
-
+* **promtail:** fix parser for azureeventhubs message without time field ([#14218](https://github.com/grafana/loki/pull/14218))
```

Comment:
it looks like this edits the changelog for `v3.1.1` which is a tagged release that has already been published.

I think this should be added to a new section called `## Main/Unreleased` or something like that.

Reply:
Done